src/renderer/components/hotbar/hotbar-icon.tsx

```diff
@@ -67,7 +67,10 @@ function onMenuItemClick(menuItem: CatalogEntityContextMenu) {
   }
 }
 
-function getNameParts(name: string): string[] {
+function getNameParts(name?: string) {
+  if (!name) {
+    return [];
```

Comment:
not sure if I should return `["", "", ""]` instead for https://github.com/lensapp/lens/blob/e97b30dae06758dc9381b551cea91d22bc7b2d08/src/renderer/components/hotbar/hotbar-icon.tsx#L99

Reply:
Why is name undefined? Should we be doing some better checks when items get added to the catalog?

Reply:
don't know, should we fix it at source instead?